### PR TITLE
Updated broken links in Metabot docs

### PR DESCRIPTION
**Pull request by sam-silv**
Quick fix of broken links to point to the right page for Pulses.

### CODE REVIEW

**Comment by samhogg** [comment]
CLA signed 😃

**Comment by kdoh** [comment]
Hey @samhogg. Thanks a ton for catching this and signing the CLA. One other dumb thing we were doing here is linking to full URLS  in the two spots you updated. If you wouldn't mind updating those to just reference ``09-pulses.md`` instead of the full urls then this should be good to merge. There's also a reference to setting up slack on line 10 that could use the same treatment if you don't mind quickly doing that while you're in there. Thanks again!

**Comment by samhogg** [comment]
@kdoh sure thing!

File: docs/users-guide/10-metabot.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Getting answers in Slack with Metabot
 
-You can already send data to Slack on  a set schedule via [Pulses](http://www.metabase.com/docs/latest/users-guide/07-pulses) but what about when you need an answer right now? Say hello to Metabot.
+You can already send data to Slack on  a set schedule via [Pulses](09-pulses) but what about when you need an answer right now? Say hello to Metabot.
```
Comment (kdoh) [inline docs/users-guide/10-metabot.md]: Unfortunately it needs the .md extension here.

File: docs/users-guide/10-metabot.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ If you don’t have a sense of which questions you want to view in  Slack, you c
 
 ## To review
 
-- [Connect to Slack](http://www.metabase.com/docs/latest/users-guide/07-pulses) to start using Metabot.
+- [Connect to Slack](09-pulses) to start using Metabot.
```
Comment (kdoh) [inline docs/users-guide/10-metabot.md]: Same here.

**Comment by samhogg** [comment]
Thanks for the tip! Updated again

**Pull request merged**